test/e2e/crd.go

```diff
@@ -209,8 +209,23 @@ overrides:
 		return targetObj, overrides, nil
 	}
 
-	validateCrud(f, tl, concreteTypeConfig, testObjectsFunc)
-
+	crudTester, targetObject, overrides := initCrudTest(f, tl, typeConfig, testObjectsFunc)
+	// Make a copy for use in the orphan check.
+	deletionTargetObject := targetObject.DeepCopy()
+	crudTester.CheckLifecycle(targetObject, overrides)
+
+	if namespaced {
+		// This check should not fail so long as the main test loop
+		// skips for limited scope.  If it does fail, manual cleanup
+		// after CheckDelete may need to be implemented.
+		if framework.TestContext.LimitedScope {
+			tl.Fatalf("Test of orphaned deletion assumes deletion of the containing namespace")
+		}
+		// Perform a check of orphan deletion.
+		fedObject := crudTester.CheckCreate(deletionTargetObject, nil)
+		orphanDeletion := true
+		crudTester.CheckDelete(fedObject, orphanDeletion)
```

Comment:
It is minor but I think there is no need for variable `orphanDeletion` because we deleted related part in this diff but keep it here.

Reply:
'true' doesn't provide any indication of my intent unless someone is familiar with the signature of `CheckDelete`, so for clarity I am declaring a variable whose name indicates my intent.  Greater programmers than me have suggested that code's primary purpose is to be read, and secondary purpose to be executed, and this is an example of emphasizing the former purpose.

Reply:
I agree on the point for code readability. Code should make itself clear.

My question is, did this work only for bool `true` but not for `false` because I found your deletion of
 - https://github.com/kubernetes-sigs/federation-v2/pull/669/files#diff-42695a0efcb8096fdb6a618bb1d79dd0R97
 - https://github.com/kubernetes-sigs/federation-v2/pull/669/files#diff-09cefc57af0991da6c524792e4183f3aR95

Reply:
The only reason `orphanDeletion` appeared everywhere before is because the parameter was being passed as a pointer to a bool.  It's not possible to take the pointer to a literal, so it was necessary to create a variable.  The only time I thought it important enough to retain the use of a variable was for the non-default 'orphan' case.  I think, though, that this is a nitty thing to worry about.

Reply:
We can close this discussion. It is not an issue. I tried to understand related consideration and learn from the code. Though the last comments does not convince me :-) for the change, it is fine for current code to me.